tests/lora/test_lora_layers_sd3.py

```diff
@@ -12,377 +12,471 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
 import sys
-import tempfile
 import unittest
 
-import numpy as np
-import torch
-from transformers import AutoTokenizer, CLIPTextConfig, CLIPTextModelWithProjection, CLIPTokenizer, T5EncoderModel
-
 from diffusers import (
-    AutoencoderKL,
     FlowMatchEulerDiscreteScheduler,
-    SD3Transformer2DModel,
     StableDiffusion3Pipeline,
 )
 from diffusers.utils.testing_utils import is_peft_available, require_peft_backend, require_torch_gpu, torch_device
 
 
 if is_peft_available():
-    from peft import LoraConfig
-    from peft.utils import get_peft_model_state_dict
+    pass
 
 sys.path.append(".")
 
-from utils import check_if_lora_correctly_set  # noqa: E402
+from utils import PeftLoraLoaderMixinTests  # noqa: E402
 
 
 @require_peft_backend
-class SD3LoRATests(unittest.TestCase):
+class SD3LoRATests(unittest.TestCase, PeftLoraLoaderMixinTests):
     pipeline_class = StableDiffusion3Pipeline
-
-    def get_dummy_components(self):
```

Comment:
No need as we more thoroughly cover all kinds of tests in `PeftLoraLoaderMixinTests` already.